src/layer/LayerGroup.js

```diff
@@ -147,6 +148,23 @@ export const LayerGroup = Layer.extend({
 	// Returns the internal ID for a layer
 	getLayerId(layer) {
 		return Util.stamp(layer);
+	},
+
+	// @method getBounds(): LatLngBounds
+	// Returns the LatLngBounds of the Layer Group (created from bounds and coordinates of its children).
+	getBounds(visitedIds = new Set()) {
+		const bounds = toLatLngBounds();
+
+		for (const layerId in this._layers) {
+			const layer = this._layers[layerId];
+			if (visitedIds.has(layerId)) {
+				continue;
+			}
+
+			visitedIds.add(layerId);
+			bounds.extend(layer instanceof LayerGroup ? layer.getBounds(visitedIds) : layer.getLatLng());

```

Comment:
Not all layers have `.getLatLng()`, so we need to check for `getBounds()` (ImageOverlay) and maybe `getLatLngs()` too. I would add also a check for `getLatLng`, because maybe a layer is included with no coordinates.

```suggestion
if(layer instanceof LayerGroup || layer.getBounds){
			bounds.extend(layer.getBounds(visitedIds));
} else if(layer.getLatLngs){
			bounds.extend(layer.getLatLngs());
} else if(layer.getLatLng){
			bounds.extend(layer.getLatLng());
}
```
bounds.extend(layer.getBounds ? layer.getBounds() : layer.getLatLng());